kyc-levels.md

```diff
@@ -18,6 +18,7 @@ Fractal ID supports different levels of verification. Additionally, each level m
 | `video` | Identity check using real time video call \(Video ID\). |
 | `accreditation` | Accreditated investor check; applicable to US and Canadian residents only. |
 | `wallet` | Crypto currency wallet check; uses [Coinfirm](https://www.coinfirm.com/). |
+| `sow` | Source of wealth check. |
```

Comment:
We should probably add `sow` to add-on matrix too, right?

Reply:
Yes, thanks for noticing. I grepped `ssn` but somehow missed that one

Reply:
Should we mention source of funds here too. Sales team is selling this as `source of funds` 🤔

Reply:
https://trustfractal.slack.com/archives/CEMTWQ411/p1561641856026800